packages/bigquery/benchmark/bench.js

```diff
@@ -0,0 +1,69 @@
+/*!
+ * Copyright 2014 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const async = require('async');
+const fs = require('fs');
+const util = require('util');
+const BigQuery = require('../src/index.js');
+
+if (process.argv.length < 3) {
+  throw util.format("need query file; usage: '%s %s <queries.json>'", process.argv[0], process.argv[1]);
```

Comment:
We could just use an es6 template string here

```js
throw `need query file; usage: '${process.argv[0]} ${process.argv[1]} <queries.json>'`;
```

Reply:
Done